### ISSUE TITLE
SCALA tuning files issue?

### ISSUE THREAD

**Issue by denniskulpa** [TomWhitwell/RM_Chorder#18]
I just finished building the Chord Organ, loaded Ver. 0.6 of the firmware and everything appears to work properly.  I then downloaded the SCALA scales.zip file, and followed the instructions to copy a new scale file onto the SD card.  I don't hear any dramatic improvements/changes to the chords played.  I've tried several different files but regardless which SCALA file I put onto the card, I can't perceive any difference.  What am I missing here??

**Comment by Normalised** [comment]
So you followed the instructions exactly as on the release page :

1.     LOAD the 0.6 Beta Firmware (Chord_Organ_0.6.hex)
2.     find an scl file you want to use, e.g. riley_albion.scl
3.     copy it to the SD card and rename it to TUNING.SCL
4.     restart your chord organ

I would recommend trying something like dudon_crying_commas.scl from the Scala Tuning Archive : http://www.huygens-fokker.org/docs/scales.zip

As its quite obvious when it is working.

You might want to double check the TUNING.SCL file when its copied to the SD card to make sure the contents are correct. SCL files are just text files, so open it with any text editor.

If that's still not working then verify that it is 0.6 firmware by testing the long press waveform to reboot :

> If you hold the WAVEFORM button for over 5 seconds the module will reboot

And then if you can paste here any options you have in CHORDORG.TXT on the SD card.

thanks.

**Comment by denniskulpa** [comment]
Thanks for the follow-up.  I tried your suggestions and I'm still not getting anywhere.

I downloaded the firmware and configuration file here:
http://polyfather.com/chord_organ/

The firmware read as 0.6 in Teensy Loader and I verified it with the long hold of the WAVEFORM button to reboot.

Here's the contents of the config file:

o   o o   o  o-o  o-O-o   o-o   o-O-o o  o o-O-o o   o  o-o 
|\ /| |   | |       |    /        |   |  |   |   |\  | o
| O | |   |  o-o    |   O         |   O--O   |   | \ | |  -o
|   | |   |     |   |    \        |   |  |   |   |  \| o   |
o   o  o-o  o--o  o-O-o   o-o     o   o  o o-O-o o   o  o-o

  o-o o  o  o-o  o--o  o-o        o-o  o--o   o-o    O  o   o
 /    |  | o   o |   | |  \      o   o |   | o      / \ |\  |
O     O--O |   | O-Oo  |   O     |   | O-Oo  |  -o o---o| \ |
 \    |  | o   o |  \  |  /      o   o |  \  o   | |   ||  \|
  o-o o  o  o-o  o   o o-o        o-o  o   o  o-o  o   oo   o

# CONFIG / FIRMWARE VERSION 0.6 #

1 [0,4,7,12,0]
2 [0,3,7,12,0]
3 [0,4,7,11,0]
4 [0,3,7,10,0]
5 [0,4,7,11,14]
6 [0,3,7,10,14]
7 [0,5,7,12,0]
8 [0,7,12,0,7]
9 [0,5,12,0,5]
10 [0,4,7,8,0]
11 [0,3,7,8,0]
12 [0,3,6,0,3]
13 [0,4,8,0,4]
14 [0,0,0,0,0]
15 [-12,-12,0,0,0]
16 [-12,0,0,12,24]

And here's the contents of the .SCL file:
! dudon_crying_commas.scl
!
Pentatonic differentiallly-coherent scale with crying commas
! Dudon 2007
 12
!
 9/8
 217/192
 437/384
 55/48
 4/3
 3/2
 3/2
 5/3
 323/192
 163/96
 55/32
 2/1

And FWIW, I'm using Teensy Loader 1.40 on a MacMini computer running Sierra 10.12.6.  I tried 2 different SD cards and got the same results.

**Comment by Thonk-Steve** [comment]
Hi,

Could you post some audio of how it sounds using the
dudon_crying_commas.scl ?

Thanks
steve

**Comment by denniskulpa** [comment]
I can record some audio.  Is there any particular wave shape (sine, square, etc.) or chord selection where the difference would be most notable or is it something that will be "intuitively obvious to the casual observer" regardless of the module settings?

**Comment by denniskulpa** [comment]
I figured it out - I THINK!  Long story short, I wasn't listening to the module correctly so I wasn't hearing the tonal differences.

Long story:
After Steve's request to post some audio and my question about what settings to use, I got to thinking.  I've only been playing with modular for about 2 years but one thing that I've learned/observed is that some sounds will sound (??) different depending on how they're presented - waveshape, filtration, chords/notes used, tempo, arrpegiation, etc.  For my initial tests, I was using single notes or slow short sequences, mostly with the sine-wave shape.

The first thing that I did, I also have Radio Music that I built some time ago, I loaded the Chord Organ firmware onto that and tried the same tests, got the same results; this told me that the hardware was OK, otherwise I had two disfunctional builds.

So I went back to the Chord Organ, fed a random sequence from a Doepfer A-149-1 into the ROOT input and a slow A/D envelope into the CHORD input, set the waveshape to SAWTOOTH.  I used two different SD cards, one with just the configuration file on it and the second had the config file and a tuning.scl file.  I then ran the sequence, starting with the straight config file, then I would swap cards on the fly - remove the first card, plug in the second, then reboot with the WAVEFORM button.  Doing this quickly, with the settings used, it was obvious to hear the tonal differences - at least with SOME of the SCL files.  The dudon_crying_commas.scl was noticable, yes there is a difference.  Some of the others were also good, with others the differences were very subtle - I guess with 4000+ different scales, it's difficult to get them all to be unique.

It's a GREAT module; even with just the standard configuration there's a lot that can be done.  GOOD job for all involved, hardware and software.

THANKS,
Dennis
